.github/workflows/continuous-integration-workflow.yml

```diff
@@ -77,9 +77,10 @@ jobs:
               ctest --test-dir builddir --output-on-failure"
       - name: Push the image into GitHub Container Registry
         uses: docker/build-push-action@v6
-        if: ${{ github.repository_owner == 'kokkos' && ( github.event_name == 'push' ||  github.event_name == 'schedule' ) }}
```

Comment:
We could discuss to add this back, the step is relatively cheap except for the intel container and I prefer to run the whole CI as part of any PR.